src/index.tsx

```diff
@@ -86,19 +86,16 @@ const MuiTelInput = React.forwardRef(
       }
     }
 
-    const focusInputElement = (): void => {
-      if (inputRef.current) {
-        inputRef.current.focus()
-      }
-    }
-
-    const handleChangeCountry = (newCountry: MuiTelInputCountry): void => {
-      setAnchorEl(null)
-      onCountryChange(newCountry)
-      if (focusOnSelectCountry) {
-        focusInputElement()
-      }
-    }
+    const handleChangeCountry = React.useCallback(
```

Comment:
I don't understand why this is a `useCallback` as the definition of `onCountryChange` is always different.

What the purpose of memoizing the function ?

Reply:
To be honested, I also thought there is no point in the `useState` and `useCallback`. The `useState` hack causes other issues and the `useCallback` seems to be unneeded as it always will rerender